From 841fe73f58f58341fcadadc1c8d58e9c8d27d546 Mon Sep 17 00:00:00 2001
From: Sebastian Pipping <sebastian@pipping.org>
Date: Sat, 9 Feb 2013 04:55:45 +0100
Subject: [PATCH] Fix "mixed implicit and normal rules" with GNU make >=3.82

---
 lightcrafts/mk/recurse.mk | 13 ++++++++++++-
 1 file changed, 12 insertions(+), 1 deletion(-)

diff --git a/lightcrafts/mk/recurse.mk b/lightcrafts/mk/recurse.mk
index 9f0d43f..ce32153 100644
--- a/lightcrafts/mk/recurse.mk
+++ b/lightcrafts/mk/recurse.mk
@@ -20,7 +20,18 @@ endif
 # overly verbose.
 
 .PHONY: all
-all %:
+all:
+	@for dir in $(SUBDIRS); \
+	do \
+	    if [ -f $$dir/*akefile ]; \
+	    then \
+		echo "-----> Entering $$dir"; \
+	        $(MAKE) --no-print-directory -C $$dir $@ || exit 1; \
+		echo "-----> Leaving $$dir"; \
+	    fi; \
+	done
+
+%:
 	@for dir in $(SUBDIRS); \
 	do \
 	    if [ -f $$dir/*akefile ]; \
-- 
1.8.1.2

