From b7f0b11552145a1e038ee02520b2b4a6525d9d8a Mon Sep 17 00:00:00 2001
From: Sebastian Pipping <sebastian@pipping.org>
Date: Sat, 9 Feb 2013 05:22:24 +0100
Subject: [PATCH 6/6] Quickfix compile flags

---
 lightcrafts/mk/platform.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lightcrafts/mk/platform.mk b/lightcrafts/mk/platform.mk
index f8700ce..4d1240e 100644
--- a/lightcrafts/mk/platform.mk
+++ b/lightcrafts/mk/platform.mk
@@ -167,7 +167,7 @@ else
   #
   # See Sun bug ID: http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=5102720
   ##
-  P4_CPU_FLAGS:=	-march=pentium4
+  P4_CPU_FLAGS:=	-march=native
 
   SSE_FLAGS_OFF:=	$(P4_CPU_FLAGS) -mno-sse
   SSE_FLAGS_ON:=	$(P4_CPU_FLAGS) -msse2
@@ -236,7 +236,7 @@ endif
 # Linux
 ##
 ifeq ($(PLATFORM),Linux)
-  PLATFORM_CFLAGS+=	-march=pentium3 -mtune=pentium4 $(SSE_FLAGS_ON) -fpermissive
+  PLATFORM_CFLAGS+=	-march=native -fPIC
   ifdef HIGH_PERFORMANCE
     PLATFORM_CFLAGS+=	-O3
   else
-- 
1.8.1.2

