From d80ffc8a952cabdbbfccd3746eec7b9a267348b7 Mon Sep 17 00:00:00 2001
From: Sebastian Pipping <sebastian@pipping.org>
Date: Sat, 9 Feb 2013 05:18:12 +0100
Subject: [PATCH 4/6] Stop using _mm_srai_epi64

---
 lightcrafts/jnisrc/macstl/impl/vec_mmx.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lightcrafts/jnisrc/macstl/impl/vec_mmx.h b/lightcrafts/jnisrc/macstl/impl/vec_mmx.h
index d46b491..3b91bf2 100644
--- a/lightcrafts/jnisrc/macstl/impl/vec_mmx.h
+++ b/lightcrafts/jnisrc/macstl/impl/vec_mmx.h
@@ -2776,7 +2776,6 @@ template <> struct FN##_function <ARG1*, ARG2 >													\
 
 				DEFINE_MMX_UNARY_FUNCTION_WITH_LITERAL (srai, _mm_srai_epi16, M128I_I16, M128I_I16)
 				DEFINE_MMX_UNARY_FUNCTION_WITH_LITERAL (srai, _mm_srai_epi32, M128I_I32, M128I_I32)
-				DEFINE_MMX_UNARY_FUNCTION_WITH_LITERAL (srai, _mm_srai_epi64, M128I_I64, M128I_I64)
 
 				DEFINE_MMX_BINARY_FUNCTION (srl, _mm_srl_epi16, M128I_I16, M128I_U16, M128I_I16)
 				DEFINE_MMX_BINARY_FUNCTION (srl, _mm_srl_epi16, M128I_U16, M128I_U16, M128I_U16)
-- 
1.8.1.2

