--- CMakeLists.txt~	2012-11-14 11:07:19.000000000 +0100
+++ CMakeLists.txt	2012-11-14 11:10:38.586774103 +0100
@@ -474,13 +470,6 @@
 
     install(
         FILES
-        ../build/GG/CMakeCache.txt
-        DESTINATION build/GG
-        COMPONENT COMPONENT_FREEORION_BUILD_FILES_GIGI
-    )
-
-    install(
-        FILES
         ../build/CMakeCache.txt
         DESTINATION build
         COMPONENT COMPONENT_FREEORION_BUILD_FILES_FREEORION
@@ -590,21 +579,6 @@
         StrCpy $7 $INSTDIR
         \\\${WordReplace} $7 \\\"\\\\\\\" \\\"/\\\" \\\"+\\\" $8
 
-	Rename $INSTDIR\\\\build\\\\GG\\\\CMakeCache.txt $INSTDIR\\\\build\\\\GG\\\\CMakeCache.txt.in
-        FileOpen $0 $INSTDIR\\\\build\\\\GG\\\\CMakeCache.txt.in r
-        FileOpen $5 $INSTDIR\\\\build\\\\GG\\\\CMakeCache.txt w
-        ClearErrors
-        loop1:
-            FileRead $0 $1
-            IfErrors done1
-            \\\${WordReplace} $1 \\\"${sdk_root}\\\" $8 \\\"+\\\" $2
-            FileWrite $5 $2
-            Goto loop1
-        done1:
-        FileClose $0
-        FileClose $5
-        Delete $INSTDIR\\\\build\\\\GG\\\\CMakeCache.txt.in
-
 	Rename $INSTDIR\\\\build\\\\CMakeCache.txt $INSTDIR\\\\build\\\\CMakeCache.txt.in
         FileOpen $0 $INSTDIR\\\\build\\\\CMakeCache.txt.in r
         FileOpen $5 $INSTDIR\\\\build\\\\CMakeCache.txt w
