diff --git a/engine/core/video/image.cpp b/engine/core/video/image.cpp
index a47db41..c3edb60 100644
--- a/engine/core/video/image.cpp
+++ b/engine/core/video/image.cpp
@@ -26,6 +26,7 @@
 
 // 3rd party library includes
 #include <SDL.h>
+#include <png.h>
 
 // FIFE includes
 // These includes are split up in two parts, separated by one empty line
diff --git a/engine/core/video/image.h b/engine/core/video/image.h
index 54c92f5..b166ad5 100644
--- a/engine/core/video/image.h
+++ b/engine/core/video/image.h
@@ -28,7 +28,6 @@
 // 3rd party library includes
 #include <SDL.h>
 #define PNG_SKIP_SETJMP_CHECK
-#include <png.h>
 
 // FIFE includes
 // These includes are split up in two parts, separated by one empty line
