diff --git a/3dfsb.c b/3dfsb.c
index 18417b3..63f3db8 100644
--- a/3dfsb.c
+++ b/3dfsb.c
@@ -42,28 +42,21 @@
 #include "config.h"
 #endif
 
-#ifdef WIN32
-#include <windows.h>
-#endif
-
 #include <GL/gl.h>
 #include "SDL/SDL.h"
 #include "SDL/SDL_opengl.h"
 
-#ifndef WIN32
 #include <GL/glx.h>
 #include "SDL/SDL_syswm.h"
-#include <gst/gl/x11/gstgldisplay_x11.h>
-#endif
 
 #include <gst/gst.h>
-#include <gst/gl/gl.h>
-
-// For saving pixbuf's to a file
-#include <gtk/gtk.h>
 
 #include <magic.h>
 
+// For saving pixbuf's to a file for debugging, not needed currently
+//#include <gtk/gtk.h>
+
+
 #ifndef _GLUfuncptr
 #define _GLUfuncptr void*
 #endif
